superset-frontend/src/explore/components/useExploreAdditionalActionsMenu/index.jsx

```diff
@@ -246,14 +249,25 @@ export const useExploreAdditionalActionsMenu = (
         openKeys={openSubmenus}
         onOpenChange={setOpenSubmenus}
       >
-        {slice && (
-          <>
+        <>
+          {slice && (
             <Menu.Item key={MENU_KEYS.EDIT_PROPERTIES}>
               {t('Edit chart properties')}
             </Menu.Item>
-            <Menu.Divider />
-          </>
-        )}
+          )}
+          {isFeatureEnabled(FeatureFlag.CROSS_REFERENCES) && (
```

Comment:
this should be behind the `slice &&` check too. If you create a new chart, slice is undefined before saving, so there's no point in showing "Dashboards added to" because it's always empty. If you do that, you can also make `chartId` required instead of optional in types of `DashboardSubMenu`

Reply:
The design specification shows the "Dashboards added to" with a `None` item. I think this is intentional to present the user with the same experience when there's no dashboard associated whether is a new chart or a saved one.

<img width="442" alt="Screen Shot 2022-10-06 at 1 43 13 PM" src="https://user-images.githubusercontent.com/70410625/194371000-b1fa250f-1f77-4842-b9d4-0917ec5b9bc1.png">